docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,182 @@
+
+Prerequisite: ``pip install mlflow==3.0.0.rc0 openai``
+
+```python
+from openai import OpenAI
+
+import mlflow
+from mlflow.metrics.genai import answer_correctness, answer_similarity, faithfulness
+
+# turn on autologging for automatic tracing
+mlflow.openai.autolog()
+
+client = OpenAI()
+
+mlflow_ground_truth = (
+    "MLflow is an open-source platform for managing "
+    "the end-to-end machine learning (ML) lifecycle. It was developed by Databricks, "
+    "a company that specializes in big data and machine learning solutions. MLflow is "
+    "designed to address the challenges that data scientists and machine learning "
+    "engineers face when developing, training, and deploying machine learning models."
+)
+# Define evaluation metrics
+metrics = {
+    "answer_similarity": answer_similarity(model="openai:/gpt-4o"),
+    "answer_correctness": answer_correctness(model="openai:/gpt-4o"),
+    "faithfulness": faithfulness(model="openai:/gpt-4o"),
+}
+question = "What is MLflow?"
+
+with mlflow.start_run():
+    response = (
+        client.chat.completions.create(
+            messages=[{"role": "user", "content": question}],
+            model="gpt-4o-mini",
+            temperature=0.1,
+            max_tokens=2000,
+        )
+        .choices[0]
+        .message.content
+    )
+
+    # Calculate metrics based on the input, response and ground truth
+    # The evaluation metrics are callables that can be invoked directly
+    answer_similarity_score = metrics["answer_similarity"](
+        predictions=response, inputs=question, targets=mlflow_ground_truth
+    ).scores[0]
+    answer_correctness_score = metrics["answer_correctness"](
+        predictions=response, inputs=question, targets=mlflow_ground_truth
+    ).scores[0]
+    faithfulness_score = metrics["faithfulness"](
+        predictions=response, inputs=question, context=mlflow_ground_truth
+    ).scores[0]
+
+    # Fetch the LoggedModel that's automatically created during autologging
+    logged_model = mlflow.last_logged_model()
+    # Log metrics and pass model_id to link the metrics
+    mlflow.log_metrics(
+        {
+            "answer_similarity": answer_similarity_score,
+            "answer_correctness": answer_correctness_score,
+            "faithfulness": faithfulness_score,
+        },
+        model_id=logged_model.model_id,
+    )
+
+
+print(f"LoggedModel model id: {logged_model.model_id}")
+# LoggedModel model id: a208e70b-e80b-4f8e-b210-20737faadd20
+
+traces = mlflow.search_traces(model_id=logged_model.model_id)
+print(traces)
+#                          request_id                                              trace  ...                                               tags assessments
+# 0  5882df1240cf4dbf845fdc9fa26c4168  Trace(request_id=5882df1240cf4dbf845fdc9fa26c4...  ...  {'mlflow.artifactLocation': 'file:///Users/ser...          []
+# [1 rows x 11 columns]
+```
+
+Navigate to the run's page to view the LoggedModel:
```

Comment:
We should show the model's page first and explain that it shows metrics from all evaluation jobs. At the *end*, we should say that users can navigate navigate to an individual evaluation job by clicking the link from the model page & we should show the runs page *last*